arch/x86/core/float.c

```diff
@@ -57,7 +57,7 @@ extern u32_t _sse_mxcsr_default_value;
  * specified thread control block. The SSE registers are saved only if the
  * thread is actually using them.
  */
-static void _FpCtxSave(struct tcs *tcs)
```

Comment:
We should rename tcs variable in this file to something else

Reply:
ok, I can change the variable name.